src/libs/runtime-utils.js

```diff
@@ -10,7 +10,7 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-export const DEFAULT_DISK_SIZE = 50
+export const DEFAULT_DISK_SIZE = 60
 export const DEFAULT_BOOT_DISK_SIZE = 50
```

Comment:
The 60G limitation may actually be with the boot disk, so I wonder if that const needs to change instead.

Actually I'm confused how `DEFAULT_DISK_SIZE` and `DEFAULT_BOOT_DISK_SIZE` are actually used in the UI. Only diskSize should actually be changeable by the user -- the boot disk should be constant. Maybe will look at UI code a bit more to understand better.

Reply:
Oh yea I see - I'll fix it